astroplan/constraints.py

```diff
@@ -402,8 +402,8 @@ class DarkGreyBrightConstraint(Constraint):
     """
     # TODO: make a new test
     @u.quantity_input(horizon=u.deg)
-    def __init__(self, lunar_horizon=90*u.deg, min_moon_illumination=0,
-                                                max_moon_illumination=1):
+    def __init__(self, lunar_horizon=90*u.deg, min_moon_illumination=None,
```

Comment:
"Lunar horizon" to me sounds like a quantity for the observer on the moon. What about "maximum lunar altitude"?